Library/Formula/cdiff.rb

```diff
@@ -0,0 +1,15 @@
+class Cdiff < Formula
+  desc "View colored, incremental diff in workspace or from stdin with side by side and auto pager support"
```

Comment:
This is likely to be too long. Limit is 80 characters including the formula name.